src/main/java/com/findmyrecycling/fmrenterprise/dto/Facility.java

```diff
@@ -34,4 +34,23 @@ public java.lang.String toString() {
                 ", facilityAddress='" + facilityAddress + '\'' +
                 '}';
     }
+    public Long getFacilityId() {
+        return facilityId;
+    }
+
+    public Long getMaterialId() {
+        return materialId;
+    }
+
+    public String getFacilityName() {
+        return facilityName;
+    }
+
+    public String getFacilityPhotos() {
+        return facilityPhotos;
+    }
+
+    public String getFacilityAddress() {
+        return facilityAddress;
+    }
```

Comment:
These are not needed, as the class is already annotated with @Data.  Remove these.